e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -683,6 +683,45 @@ describe("scenarios > dashboard", () => {
       cy.findByText("There was a problem displaying this chart.");
     });
   });
+
+  it("should now overflow markdown content in description (metabase#31326)", () => {
+    cy.intercept("GET", "/api/dashboard/1").as("getDashboard");
+    cy.intercept("PUT", "/api/dashboard/1").as("updateDashboard");
+    visitDashboard(1);
+    cy.wait("@getDashboard");
+
+    cy.get("main header").within(() => {
+      cy.icon("info").click();
+    });
```

Comment:
nit: We could use [`icon`](https://github.com/metabase/metabase/blob/master/e2e/support/commands/ui/icon.js) right after a subject now

```suggestion
    cy.get("main header").icon("info").click();
```